terraform/modules/private-dns-zone-extend/variables.tf

```diff
@@ -0,0 +1,8 @@
+#Variable for passingaccount name to module for dns lookup
```

Comment:
You can remove the comments from the variables and supply them as a `description = "example"` in each variable block

Reply:
removed and description added